### PR TITLE
fmt, testing fixes and helpers

### PR DESCRIPTION
**Pull request by sjhx**
A few things going on here:

1. fix e2e tests for compatibility with K1.25 (cronjob type)
2. some scripts to help run tests 
3. Makefile updates to make fmt be more useful and tests to be safer

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-GOFILES=$(shell find . -type f -name '*.go' -not -path "./code-generator/*")
+GOFILES=$(shell find . -type f -name '*.go' -not -path "./code-generator/*"i -not -path "./pkg/apis/*")
```
Comment (tim-gp) [inline Makefile:1]: this looks a bit strange

File: Makefile [hunk 2]
```diff
@@ -20,15 +20,16 @@ test-deps:
...
-	@${GOPATH}/src/github.com/IBM/portieris/scripts/copyright.sh
+	./scripts/copyright.sh
 
 copyright-check:
-	@${GOPATH}/src/github.com/IBM/portieris/scripts/copyright-check.sh
+	./scripts/copyright-check.sh
 
 fmt:
+	@gofmt -l ${GOFILES}
 	@if [ -n "$$(gofmt -l ${GOFILES})" ]; then echo 'Please run gofmt -l -w on your code.' && exit 1; fi
```
Comment (tim-gp) [inline Makefile:33]: this line probably not needed now?

File: Makefile [hunk 3]
```diff
@@ -62,35 +63,37 @@ e2e.local.ics: helm.install.local e2e.quick.ics
 
 e2e.quick: e2e.quick.trust.imagepolicy e2e.quick.trust.clusterimagepolicy e2e.quick.wildcards e2e.quick.generic e2e.quick.simple.imagepolicy e2e.quick.vulnerability
 e2e.quick.ics: e2e.quick.trust.imagepolicy e2e.quick.trust.clusterimagepolicy e2e.quick.armada e2e.quick.wildcards e2e.quick.generic e2e.quick.simple.imagepolicy
-	-kubectl delete namespace $$(kubectl get namespaces | grep -v ibm | grep -v kube | grep -v default | grep -v portieris | awk '{ print $$1 }' | grep -v NAME)
+	-kubectl delete namespace $$(kubectl get namespaces | grep '[0-9a-f]*-[0-9a-f]*-[0-9a-f]*-[0-9a-f]*-[0-9a-f]*' | awk '{ print $$1 }' )
```
Comment (tim-gp) [inline Makefile:66]: delete everything? 💀

**Review by tim-gp — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (sjhx) [reply Makefile:33]: why?
first line lists the files with fmt issues 
second line tells you how to fix it

File: Makefile (hunk 3 above)
Comment (sjhx) [reply Makefile:66]: rather than excluding known namespaces from the delete everything it now matches the uuid format names

**Review by tim-gp — APPROVED** [review]
LGTM

**Pull request merged**